Code/GraphMol/Substruct/catch_tests.cpp

```diff
@@ -15,11 +17,49 @@
 
 #include <GraphMol/RDKitBase.h>
 #include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/SmilesParse/SmilesWrite.h>
 #include <GraphMol/Substruct/SubstructMatch.h>
 
 using namespace RDKit;
 typedef std::tuple<std::string, std::string, size_t> matchCase;
 
+class _IsSubstructOf : public Catch::MatcherBase<const ROMol &> {
```

Comment:
oooo, taking advantage of some of the extra stuff catch2 provides.
Nice!

Reply:
👍